ui/lib/core/addon/decorators/fetch-secrets-engine-config.ts

```diff
@@ -40,7 +40,10 @@ export function withConfig(modelName: string) {
 
       async beforeModel(transition: Transition) {
         super.beforeModel(transition);
-
+        if (!this.secretMountPath) {
+          // eslint-disable-next-line
+          console.error('secretMountPath service is required for withConfig decorator. Add it to the route');
+        }
```

Comment:
I think we should avoid using console.error, can we update this to either `console.debug` or use `assert` 
```suggestion
        if (!this.secretMountPath) {
          // eslint-disable-next-line
          console.debug('secretMountPath service is required for withConfig decorator. Add it to the route');
        }
```

Reply:
`import { assert } from '@ember/debug';`    
```suggestion
      assert('secretMountPath service is required in routes using the @withConfig decorator, this.secretMountPath);
```